CMakeLists.txt

```diff
@@ -12,6 +12,10 @@ include(${CMAKE_CURRENT_LIST_DIR}/cmake/utils.cmake)
 #
 set(PYTHON_SUPPORTED_VERSIONS "3.8" "3.9" "3.10" "3.11")
 
+# used when building pytorch-related extensions
+# TODO(youkaichao): only compute this if we are building wheels
+# TODO(youkaichao): otherwise autodetect the version from current GPUs
+set(TORCH_CUDA_ARCH_LIST "7.0;7.5;8.0;8.6;8.9;9.0")
```

Comment:
`CUDA_SUPPORTED_ARCHS` is exactly the same flag and used as a replacement  for `TORCH_CUDA_ARCH_LIST`

Reply:
`CUDA_SUPPORTED_ARCHS` is not respected during build, I find `TORCH_CUDA_ARCH_LIST` can work correctly. Otherwise, it tries to compile for 9.0a architecture, which our current ci machine's `nvcc` does not support.

Reply:
@youkaichao But how is this related this to this PR?

Reply:
Looks good to me. This happens when I try to upgrade to pt 2.2.0 , because pt 2.2.0 defines the default `TORCH_CUDA_ARCH_LIST` to contain `9.0a` .

We can remove this for current PR.

Reply:
Fixed in [0361127](https://github.com/vllm-project/vllm/pull/3625/commits/0361127364442adffaa60373beb255f168505ac2).